rules/linux/macos_susp_histfile_operations.yml

```diff
@@ -0,0 +1,29 @@
+title: 'Suspicious History File Operations'
+id: 508a9374-ad52-4789-b568-fc358def2c65
+status: experimental
+description: 'Detects commandline operations on shell history files'
+    # Rule detects presence of various shell history files in process commandline
+    # Normally user expected to view own history with dedicated 'history' command and not some other tools
+    # There is a possibility for rule to trigger, when T1070.003 techinuque is used (history file cleared)
+    # For this rule to work you must enable audit of process execution in OpenBSM, see
+    # https://osquery.readthedocs.io/en/stable/deployment/process-auditing/#macos-process-socket-auditing
+author: 'Mikhail Larin, oscd.community'
+date: 2020/10/17
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1552.003/T1552.003.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection:
+        CommandLine|contains:
+            - '.bash_history'
+            - '.zsh_history'
```

Comment:
Hey mate! You can also add `.sh_history` here (:

Reply:
Are there a lot of people using ksh in macOS? If so, i'll fix this one

Reply:
Well, actually, the list is bigger. Take a look at what the osquery is watching for in the [`shell_history` table](https://github.com/theopolis/osquery/blob/327a6a3e247efc6420ac6b541cc5ec4349e2e879/osquery/tables/system/posix/shell_history.cpp#L23-L28) (the same for both Linux and macOS).